src/index.html

```diff
@@ -84,7 +84,7 @@
                  data-chart-title="Number of loans originated"
                  data-chart-color="blue"
                  data-chart-description="This is the chart description."
-                 data-chart-source="consumer-credit-trends/num_data_AUT.csv">
+                 data-chart-source="consumer-credit-trends/auto-loan/num_data_AUT.csv">
```

Comment:
I can do this on the script side, will check with @contolini if anything needs to change in JAC